api/envoy/service/ratelimit/v2/rls.proto

```diff
@@ -8,6 +8,8 @@ import "envoy/api/v2/ratelimit/ratelimit.proto";
 
 import "validate/validate.proto";
 
+// [#protodoc-title: RateLimitService]
```

Comment:
"Rate Limit Service (RLS)"

Also, can you take a look at the generated docs? Nothing is actually showing up since things are hidden using not implemented tags. It would be great to do a small pass here just to make sure things look OK. Thank you!

Reply:
Thanks, updated it in dd13189.